app/src/main/java/com/example/cleanarchitecture/utils/Data.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.cleanarchitecture.utils
+
+data class Data<RequestData>(var responseType: Status, var data: RequestData? = null, var error: Exception? = null)
```

Comment:
as a good practice, each viewModel should have its own data and status, please put these classes in the ViewModel that use them and remove this file

Reply:
done